doc/manipulating_images/input_output.rst

```diff
@@ -234,6 +234,16 @@ data, which we call Niimgs or Niimg-4D. Accepted input arguments are:
    If you provide a sequence of Nifti images, all of them must have the same
    affine.
 
+.. topic:: Decreasing memory used when loading Nifti images
+
+   When Nifti images are stored compressed (.nii.gz), loading them directly
+   consumes more memory. As a result, large 4D images may
```

Comment:
Is it worth explaining why this is true ?

Reply:
My sense is that it may not be worth getting too much into the weeds about why.  @GaelVaroquaux ?

Reply:
I agree. Less is more